src/test/java/com/google/api/gax/grpc/SettingsTest.java

```diff
@@ -126,16 +124,15 @@
     }
 
     private final SimpleCallSettings<Integer, Integer> fakeMethodSimple;
-    private final PageStreamingCallSettings<Integer, Integer, FakePagedListResponse>
+    private final PagedCallSettings<Integer, Integer, FakePagedListResponse>
         fakeMethodPageStreaming;
     private final BundlingCallSettings<Integer, Integer> fakeMethodBundling;
 
     public SimpleCallSettings<Integer, Integer> fakeMethodSimple() {
       return fakeMethodSimple;
     }
 
-    public PageStreamingCallSettings<Integer, Integer, FakePagedListResponse>
-        fakeMethodPageStreaming() {
+    public PagedCallSettings<Integer, Integer, FakePagedListResponse> fakeMethodPageStreaming() {
```

Comment:
please do a text-based grep to make sure you find all cases where "page streaming" should be renamed.